api/serverpb/server.proto

```diff
@@ -161,6 +163,32 @@ message AWSInstanceCheckRequest {
 
 message AWSInstanceCheckResponse {}
 
+message PlatformSignUpRequest {
+  // New Percona Platform user's email.
+  string email = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  // New Percona Platform user's password.
+  string password = 2;
```

Comment:
The presence of that field should be validated in code. In the future, we will relax that requirement. Removing a validator would be a breaking change from prototool perspective.